web/src/slices/types.ts

```diff
@@ -12,8 +12,10 @@ export interface Connection {
 
 export interface ProofRequestData {
   connectionId: string
-  attributes?: ProofRequestData[]
-  predicates?: ProofRequestData[]
+  /* eslint-disable  @typescript-eslint/no-explicit-any */
+  attributes?: any[]
+  /* eslint-disable  @typescript-eslint/no-explicit-any */
+  predicates?: any[]
```

Comment:
Assume this is intentional, want to confirm.

Reply:
Intentional, the attributes and predicates arrays can be several different shapes. My instinct is to leave arrays/objects like this, as typing them will cause more problems than it solves.

Reply:
:+1:

This would be a good PR to switch back from inline no-explicit-any's to a global one in .eslintrc -- makes it easier to turn on and off to find types that need to be defined in the future.

Reply:
OK I will add that in